### PR TITLE
Features and open source - Minswap.json

### PR DESCRIPTION
**Pull request by clawoo**

### CODE REVIEW

File: dApps/Minswap.json [hunk 1]
```diff
@@ -3,7 +3,15 @@
     "link": "https://minswap.org",
     "twitter": "https://twitter.com/MinswapDEX",
     "category": "DEFI",    
-    "subCategory": "DEX",
+    "subCategory": "AMM DEX",
+    "features": ["Swap", "Limit orders", "Liquidity mining", "Yield farming"], 
+    "contracts": [
+        {
+         "openSource": "TRUE",
```
Comment (Cardano-Fans) [inline dApps/Minswap.json:10]: booleans are native, btw are you sure it is still open source?

```true```

or
```
false
```

no ""

Comment (clawoo) [reply dApps/Minswap.json]: I modified it. 
About open source, the link is for the contract version before vulnerability, latest version contracts are not open sourced yet.

File: dApps/Minswap.json [hunk 2]
```diff
@@ -3,7 +3,15 @@
     "link": "https://minswap.org",
     "twitter": "https://twitter.com/MinswapDEX",
     "category": "DEFI",    
-    "subCategory": "DEX",
+    "subCategory": "AMM DEX",
+    "features": ["Swap", "Limit orders", "Liquidity mining", "Yield farming"], 
+    "contracts": [
+        {
+         "openSource": true,
+         "contractLink": "https://github.com/CatspersCoffee/contracts",
```
Comment (Cardano-Fans) [inline dApps/Minswap.json:11]: Are you sure these are the latest versions after bug fixes?

Comment (clawoo) [reply dApps/Minswap.json:11]: they are not, they are the version before the bug fixes.

Comment (clawoo) [reply dApps/Minswap.json:11]: that's why the contractID is placed on version 1 scripts and not version 2

Comment (Cardano-Fans) [reply dApps/Minswap.json:11]: Perfect!!!

File: dApps/ADAXPRO.json [hunk 3]
```diff
@@ -3,7 +3,14 @@
     "link": "https://adax.pro",
     "twitter": "https://twitter.com/adax_pro",
     "category": "DEFI",    
-    "subCategory": "DEX", 
+    "subCategory": "ORDER BOOK DEX",
```
Comment (Cardano-Fans) [inline dApps/ADAXPRO.json:6]: We have enums here only :) Enums do not have white spaces in them.
I suggest something like: ORDER_BOOK_DEX

File: dApps/MuesliSwap.json [hunk 4]
```diff
@@ -3,7 +3,15 @@
     "link": "https://ada.muesliswap.com",
     "twitter": "https://twitter.com/MuesliSwapTeam",
     "category": "DEFI",    
-    "subCategory": "DEX", 
+    "subCategory": "ORDER_BOOK_DEX",
+    "features": ["Partial orders", "DEX Aggregator", "Milk staking"],
```
Comment (Cardano-Fans) [inline dApps/MuesliSwap.json:7]: I would also do this as enums, so

```
"features": ["PARTIAL_ORDERS", "DEX_AGGREGATOR", "MILK_STAKING"], 
```

Comment (clawoo) [reply dApps/MuesliSwap.json]: Done

File: dApps/ADAXPRO.json [hunk 5]
```diff
@@ -3,7 +3,14 @@
     "link": "https://adax.pro",
     "twitter": "https://twitter.com/adax_pro",
     "category": "DEFI",    
-    "subCategory": "DEX", 
+    "subCategory": "ORDER_BOOK_DEX",
+    "contracts": [
+        {
+         "openSource": true,
+         "contractLink": "https://github.com/adax-official/smart-contract-v1",
+         "contractID": "Jpse"
```
Comment (Cardano-Fans) [inline dApps/ADAXPRO.json:11]: I do think that version is the same as contractId, don't you think?

Comment (clawoo) [reply dApps/ADAXPRO.json:11]: ContractId is to point to the script version that is open sourced.

Comment (Cardano-Fans) [reply dApps/ADAXPRO.json:11]: I think it is the same but it is not always open source.... 

Show me an example where they would point to different values please?

Comment (clawoo) [reply dApps/ADAXPRO.json:11]: So, versions are the plutus scripts, every time they make a new script, I made a new version.
auditId is referencing to the version of the script that was audited by external auditor.
In same way contractId is referencing to the version of the script that was put as open source (on github mostly).
contractId and auditId can point to different versions of the plutus scripts.

Comment (Cardano-Fans) [reply dApps/ADAXPRO.json:11]: ok

File: dApps/Minswap.json [hunk 6]
```diff
@@ -37,6 +45,7 @@
                     "version": 1,
                     "scriptHash": "0ca50950adbb06c5c5f6833924a66ac873e43202588b6d338602d78d",
                     "contractAddress": "addr1wyx22z2s4kasd3w976pnjf9xdty88epjqfvgkmfnscpd0rg3z8y6v",
+                    "contractID": "wFhM",
```
Comment (Cardano-Fans) [inline dApps/Minswap.json:48]: Notice that we do auditId, not auditID.

Would you be so kind to change all the places with contractID to contractId ?

This is cosmetics but will be hard to change later.

Comment (clawoo) [reply dApps/Minswap.json:48]: Ok

Comment (clawoo) [reply dApps/Minswap.json]: Done

**Pull request merged**